.github/workflows/build.yml

```diff
@@ -53,21 +49,43 @@ jobs:
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
         with:
-          mask-password: 'true'
+          mask-password: "true"
+
+      - name: Set image tag in ENV
+        id: "set-image-tag"
+        run: |-
+          echo "image_tag=$(
+          if ([ "${{ github.ref_name }}" == "main" ] || [ "${{ github.ref_name }}" == "master" ]) && [ "${{ github.event_name }}" == "push" ]; then
```

Comment:
Finnes det noen måte å referere til `default-branch` i stedet for å hardkode "main" og "master" her?

Reply:
Det tror jeg ikke. Ingen verdier i denne konteksten forteller noe om det. Konseptuelt er dette det samme som i repo-workflows hvor man eksempelvis har 

```
on:
  push:
    branches:
      - master
```

Her er det jo også hardkoda på sett og vis.

Reply:
Yupp, men det er jo kode som lever tettere på valget av default-branch, siden det spesifiseres i hvert repo. 

Hvis plutselig `main` ikke er godt nok navn på default-branch lenger, tilsvarende som for `master`, så vil jo release brekke med mindre man husker å også oppdatere det her.

Ikke et stort problem! Man vil jo oppdage det, men det hadde vært digg å slippe. Er det ikke støtte for det, så tenker jeg det er godt nok som det er nå 👍

Reply:
Ja, jeg tror ikke vi kommer unna dette. Høres bra ut!